### ISSUE TITLE
freshclam fills up entire disk with *.tmp directories at times

### ISSUE THREAD

**Issue by joram1959** [Cisco-Talos/clamav#625]
We had an issue where freshclam ate up an entire disk space. Our /var/lib/clamav directory was full of "clamav-*********.tmp directories.
I can not reproduce this problem and it happened only once...searching the internet revealed numerous folks have run into this.
I read all the release notes for each version of clamav to see if  upgrading to a newer version might prevent this from ever happening again but did not see any mention of anything like this being fixed. Our current version is 0.102.3. Again, this issue appears extremely rare as we have 1000's of devices out there running clamav on Ubuntu platforms.

**Comment by ragusaa** [comment]
Hi,

We are sorry to hear that you had this issue.  I searched our ticket queue for tickets regarding this issue, and didn't find any, so I don't know if upgrading will help.  

Please let us know if it happens again, and if there is any way you can reproduce it.  I'll put in a ticket here, and we can attempt to reproduce it locally.

Thanks,
Andy

**Comment by micahsnyder** [comment]
@joram1959 you probably should upgrade, though. ClamAV 0.102.3 is [no longer supported for new security fixes](https://docs.clamav.net/faq/faq-eol.html#version-support-matrix) and is already missing some security fixes. It is also ideal if bugs can be reported only for the latest version. We generally don't backport bug fixes to older versions unless they're are critical/security issues.

**Comment by joram1959** [comment]
Thanks for responding.. I only saw this problem once and was not able to reproduce...I don't think anyone has been able to reproduce it because I read a lot of chatter from other folks who have run into this. 
    On Friday, July 1, 2022, 12:27:22 PM EDT, ragusaa ***@***.***> wrote:  
 
 


Hi,

We are sorry to hear that you had this issue. I searched our ticket queue for tickets regarding this issue, and didn't find any, so I don't know if upgrading will help.

Please let us know if it happens again, and if there is any way you can reproduce it. I'll put in a ticket here, and we can attempt to reproduce it locally.

Thanks,
Andy

**Comment by joram1959** [comment]
Appreciate you getting back to me...now Im wondering if that's why we are seeing:Wed Jun 29 14:51:31 2022 -> ^downloadFile: Unexpected response (403) from https://database.clamav.net/daily-25879.cdiff

errors everywhere now too.
    On Wednesday, July 6, 2022, 12:18:42 PM EDT, Micah Snyder ***@***.***> wrote:  
 
 


@joram1959 you probably should upgrade, though. ClamAV 0.102.3 is no longer supported for new security fixes and is already missing some security fixes. It is also ideal if bugs can be reported only for the latest version. We generally don't backport bug fixes to older versions unless they're are critical/security issues.

**Comment by micahsnyder** [comment]
@joram1959 yes that is definitely the reason for those errors. ClamAV 0.102.x is actively blocked from updating directly from clamav.net. 

There have been a number of improvements to Freshclam in the 0.103 release in particular that we really want users to have.  But in general we're being more aggressive with requiring users to use newer versions of clamav to maintain access to database updates. 

Because this issue isn't reproducible, even by you - and because it was reported for an older, unsupported version - I'm going to close this issue.  If you or others run into this issue or any other issue with a supported version, feel free to reopen this issue or report a new issue. 

Regards,
Micah
